### ISSUE TITLE
The tags (like vendors under filters) are going off-screen

### ISSUE THREAD

**Issue by tonydisera** [iobio/genepanel.iobio#156]
<img width="389" alt="screen shot 2018-07-10 at 11 20 10 am" src="https://user-images.githubusercontent.com/8420103/42526589-789fceb8-8433-11e8-9ec1-490b14bab220.png">

**Comment by adityaekawade** [comment]
Hi @tonydisera, 
This was being caused because some of the names of the vendors are very long. 
Example: Molecular Otolaryngology and Renal Research Laboratories University of Iowa Hospital and Clinics 

As a result, the text was going out of the card. I wanted to break the text when it reaches the max width, I tried using `word-break` , `overflow-wrap: break-word` properties of the css, but it didn't seem to work probably because of the close button of the v-chip. 
Instead I have used the media queries and set a max-width of the div as per the screen size and if the text is too long it will show  an ellipsis (...)

![Screen Shot 2018-07-16 at 5.04.46 PM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5a68ffc37453d30014ef4f0d/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77246455f56b7191a37df1122f4f5b595ecc3c4a4356bcbbd671f3e57862804e0ebcd8380b7dcf6f14faee824dba01ba21d94d86e7b48818d2f42f67b6d5db1063474f4dcbed5727a19141c95df54ae64557a3113765e1762431.png)
